docs/spark-qualification-tool.md

```diff
@@ -313,95 +338,184 @@ $SPARK_HOME/bin/spark-shell --jars rapids-4-spark-tools_2.12-<version>.jar
+|            |              | Duration | Duration | Opportunity |    GPU    |    GPU    |    GPU    |                    |
+|            |              |          |          |             |  Duration |  Speedup  |    Time   |                    |
+|            |              |          |          |             |           |           |   Saved   |                    |
++============+==============+==========+==========+=============+===========+===========+===========+====================+
+| appName-01 | app-ID-01-01 |    898429|    879422|       879422|  273911.92|       3.27|  624517.06|Strongly Recommended|
++------------+--------------+----------+----------+-------------+-----------+-----------+-----------+--------------------+
+| appName-02 | app-ID-02-01 |      9684|      1353|         1353|    8890.09|       1.08|      793.9|     Not Recommended|
++------------+--------------+----------+----------+-------------+-----------+-----------+-----------+--------------------+
 ```
 
-In the above example, two application event logs were analyzed. “app-20210507174503-2538” is rated higher 
-than the “app-20210507174503-1704” because the score(in the csv output) for “app-20210507174503-2538”   
-is higher than  “app-20210507174503-1704”. 
-Here the `Problematic Duration` is zero but please keep in mind that we are only able to detect certain issues. 
-This currently includes some UDFs and nested complex types.
-The tool won't catch all UDFs, and some of the UDFs can be handled with additional steps.
+In the above example, two application event logs were analyzed. “app-ID-01-01” is "_Strongly Recommended_"
+because `Estimated GPU Speedup` is ~3.27. On the other hand, the estimated acceleration running
+“app-ID-02-01” on the GPU is not high enough; hence the app is not recommended.
+
+#### Detailed App Report
 
-Please refer to [supported_ops.md](./supported_ops.md) 
-for more details on UDF.
+**1. Entire App report**
+
+The first part of the detailed report is saved as `rapids_4_spark_qualification_output.csv`. 
+The apps  are processed and ranked by the `Estimated GPU Speed-up`.
+In addition to the fields listed in the "_Report Summary_", it shows the following fields:
+
+1. SQL Dataframe Task Duration: Amount of time spent in tasks of SQL Dataframe operations.
+2. Executor CPU Time Percent: This is an estimate at how much time the tasks spent doing processing on the CPU vs waiting on IO.
+   This is not always a good indicator because sometimes the IO that is encrypted and the CPU has to do work to decrypt it,
+   so the environment you are running on needs to be taken into account.
+3. SQL Ids with Failures: SQL Ids of queries with failed jobs.
+4. Unsupported Read File Formats and Types: Looks at the Read Schema and
+   reports the file formats along with types which may not be fully supported.
+   Example: `Parquet[decimal]`, `JDBC[*]`. Note that this is based on the current version of the plugin and
```

Comment:
Nit: Can we remove Parquet[decimal] since it is supported. I think we missed it earlier.

Reply:
Done!